basic/stam/stam_exercise.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import os
+
+PATH = 1
+
+
+def open_file(directory, mode):
+    input_file = open(directory, mode)
+    return input_file
+
+
+def main():
+    try:
+        directory = sys.argv[PATH]
+        if ".txt" in directory:
+            pass
+        else:
+            print(os.listdir(directory))
+            file_name = input("Enter file name from list above: ")
+            directory += "\\" + file_name
+
+    except Exception as error_message:
```

Comment:
Way too wide of an exception - if you try to catch one - use the specific one or the class exception